src/components/ArticleCard.jsx

```diff
@@ -1,7 +1,11 @@
+import { Link } from "react-router-dom";
+
 const ArticleCard = ({ article }) => {
   return (
     <div className="article_cards">
-      <h4>{article.title}</h4>
+      <Link to={`/articles/${article.article_id}`}>
+        <h4>{article.title}</h4>
+      </Link>
       <p>Topic = {article.topic}</p>
       <p>Author = {article.author}</p>
       <p>Created at = {article.created_at}</p>
```

Comment:
Take a look at the toUTCstring method to format this date in a more readable way